contrib/pipelines/secadvisor/secadvisor.yml.default

```diff
@@ -1,4 +1,6 @@
 host_id: ""
+analyzers:
```

Comment:
I do not see where this is supposed to be used :(

Reply:
used here: https://github.com/skydive-project/skydive/blob/960a50d2c797c60fa3c656d8335dd7406ec8dd07/contrib/pipelines/secadvisor/transform.go#L97